### ISSUE TITLE
1. Incorrect role="presentation" on image, 2. there is no role associated with area-label='crop photo'

### ISSUE THREAD

**Issue by sunilrathore24** [Mawi137/ngx-image-cropper#591]
There are 2 accessibility issues present in the latest release

1. the image (.ngx-ic-source-image) has an incorrect role="presentation", this should be removed
2. the div(.ngx-ic-cropper) has an area label='crop photo' but there is no role added which is creating a minor issue in accessibility tools

please check the attached screenshot for more details

![image](https://github.com/Mawi137/ngx-image-cropper/assets/19345828/f028eba5-bdd5-4ebc-b102-e8810d42ac75)

**Comment by Mawi137** [comment]
This was done in a PR by someone else, but I don't know enough about accessibility myself to judge how it should be applied to this component.

There are more elements that got the role="presentation", are they also incorrect?\
Any suggestion to what the role should be for the div(.ngx-ic-cropper) ?
